demos/angular/saas/src/app/app.component.html

```diff
@@ -115,8 +115,10 @@ <h2>Accounts</h2>
                       </ais-highlight>
                     </td>
                     <td>
-                      <ais-highlight attribute="Website" [hit]="hit">
-                      </ais-highlight>
+                      <span *ngIf="hit.Website">
```

Comment:
Is the `span` necessary? Can't we set the `ngIf` on `ais-highlight`?

Reply:
Should be possible, also maybe the if should be on hit._highlightResult.Website

Reply:
Indeed! We're getting better at Angular all together!